packages/widget/tests/docmaps-widget.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, Page, test } from '@playwright/test'
+
+
+test('clicking button increments the count', async ({ page }) => {
+  await page.goto('/')
+
+  await expectNodeCount(page, 3)
+  await expect(page.getByRole('button', { name: 'Add 4th Node' })).toBeVisible()
+
+  await page.getByRole('button', { name: 'Add 4th Node' }).click()
+
+  await expect(page.getByRole('button', { name: 'Add 5th Node' })).toBeVisible()
```

Comment:
"getByRole" is weird terminology to me , but in general this looks great!!!

Reply:
Yeah I'm not a fan of that language either. Weird choice on their part